musig/src/util.rs

```diff
@@ -0,0 +1,16 @@
+use sha2::{Sha256, Digest};
+use rand::Rng;
+
+pub fn random_message_hash(rng: &mut impl Rng) -> Vec<u8> {
```

Comment:
It seems that this function is only used in tests. Maybe put it into the `tests` module?